reaktive/src/jvmJsCommonMain/kotlin/com/badoo/reaktive/disposable/CompositeDisposable.kt

```diff
@@ -8,13 +8,13 @@ import kotlin.jvm.Volatile
 @Suppress("EmptyDefaultConstructor")
 actual open class CompositeDisposable actual constructor() : Disposable {
 
-    private var list: MutableList<Disposable>? = null
+    private var collection: MutableCollection<Disposable>? = null
     @Volatile
     private var _isDisposed = false
     override val isDisposed: Boolean get() = _isDisposed
 
     /**
-     * Disposes the [CompositeDisposable] and all its [Disposable]s.
+     * Atomically disposes the collection and all its [Disposable]s.
      * All future [Disposable]s will be immediately disposed.
      */
     actual override fun dispose() {
```

Comment:
Just a question. What if we add kdoc to except class? Won't it be available on actual classes?
For example, if you write kdoc for interface methods, that it will be available on implementation methods.

Reply:
Kdocs from expect classes are not visible, looks like a bug. So we have to copy them to actual classes(